server/service/src/sync/test/test_data/store_preference.rs

```diff
@@ -43,7 +43,7 @@ const STORE_PREFERENCE_1: (&str, &str) = (
         "consolidateBatches": false,
         "editPrescribedQuantityOnPrescription": false,
         "chooseDiagnosisOnPrescription": false,
-        "useConsumptionAndStockFromCustomersForInternalOrders": false,
```

Comment:
I'm curious why this was here already if it wasn't being synced?

Reply:
I recall there was a reason... but escapes my mind right now... But also I like to see all the available fields in OG without having to go to OG 😆